### PR TITLE
Add three missing reports

### PR DESCRIPTION
**Pull request by wmdmgv**
- Events Triggered Summary
  - Rules Matched Summary
  - Dispute Delays in Days

### CODE REVIEW

File: spec/definitions/ReportEventsTriggeredSummary.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            - gateway-account-requested
+            - transaction-processed
+            - subscription-canceled
+            - subscription-renewed
+            - payment-card-expired
+            - payment-declined
+            - transaction-process-requested
+            - risk-score-changed
+        count:
+          description: 25th percentile
```
Comment (2e3s) [inline spec/definitions/ReportEventsTriggeredSummary.yaml]: copypaste

File: spec/paths/reports@disputes.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ get:
         - "bin"
         - "country"
         - "rebillNumber"
-        - "dunningIndex"
+        - "retryNumber"
```
Comment (2e3s) [inline spec/paths/reports@disputes.yaml]: I suggest to change the report name and path too. Get rid of dunning at all

File: spec/swagger.yaml [hunk 3]
```diff
@@ -144,6 +144,11 @@ parameters:
     in: query
     type: string
     description: The partial search of the text fields.
+  collectionCriteria:
+    name: criteria
```
Comment (2e3s) [inline spec/swagger.yaml]: Please, ensure that `collectionCriteria` is specified for:
Transaction report, Cancellation report, Cumulative subscriptions and cumulative plan subscriptions, retention value, .

File: spec/paths/reports@disputes.yaml (hunk 2 above)
Comment (wmdmgv) [inline spec/paths/reports@disputes.yaml]: Fixed

File: spec/swagger.yaml (hunk 3 above)
Comment (wmdmgv) [inline spec/swagger.yaml]: Added to.

**Pull request merged**